### ISSUE TITLE
User-select: Fields don't have labels.

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#233]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/userselect/

* The initial select element doesn't have a label, visible or otherwise
* The code block above it isn't a form field, but it feels like it should have some kind of heading above it (not really part of this bug but probably should be fixed at the same time)
*  The name and comment text fields have labels but they're not hooked up via ID and for attribute so don't get an accName.

Fix:

* Add label to the select with for attribute that points to the ID of the select
* Add for attributes to the other two labels and the  matching ID to each of the text inputs
* Probably a good idea to add a heading above the code block.

**Comment by ststimac** [comment]
I'm working on updating this demo design-wise so I'll fix these.
